elastic/integrations
# entityanalytics_okta: Add `event.original` field when `preserve_original_event` tag is set.

## Proposed commit message
Okta Entity Analytics doesn't capture full event inside raw `message` field. Add a painless script to capture the original event into `event.original` using `Preserve original event` UI option.



## Checklist


## How to test this PR locally


Running Pipeline tests should pass successfully.
Run -> `cd packages/entityanalytics_okta && elastic-package stack down && elastic-package build && elastic-package stack up --version=8.13.0 -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test pipeline --generate -v`
Returns -> 
```
--- Test results for package: entityanalytics_okta - START ---
╭──────────────────────┬─────────────┬───────────┬───────────────────────────────────────────┬────────┬──────────────╮
│ PACKAGE              │ DATA STREAM │ TEST TYPE │ TEST NAME                                 │ RESULT │ TIME ELAPSED │
├──────────────────────┼─────────────┼───────────┼───────────────────────────────────────────┼────────┼──────────────┤
│ entityanalytics_okta │ user        │ pipeline  │ (ingest pipeline warnings test-user.json) │ PASS   │ 321.542792ms │
│ entityanalytics_okta │ user        │ pipeline  │ test-user.json                            │ PASS   │  51.258333ms │
╰──────────────────────┴─────────────┴───────────┴───────────────────────────────────────────┴────────┴──────────────╯
--- Test results for package: entityanalytics_okta - END   ---
Done
```
